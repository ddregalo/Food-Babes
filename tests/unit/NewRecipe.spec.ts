import { shallowMount, mount } from '@vue/test-utils';
const NewRecipe = require('../../src/components/NewRecipe.vue');

describe('NewRecipe Component', () => {

    var wrapper: any
  
    beforeEach(() =>{
      wrapper = shallowMount(NewRecipe);
    });
  
    it('renders the correct markup', () => {
      expect(wrapper.html()).toMatchSnapshot();
    });
  
    it('has a component object', () => {
      expect(typeof NewRecipe).toBe('object');
    });
  });