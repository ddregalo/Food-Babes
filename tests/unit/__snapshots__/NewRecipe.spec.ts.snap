// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewRecipe Component renders the correct markup 1`] = `undefined`;
